packages/plugins/persisted-operations/src/index.ts

```diff
@@ -29,11 +30,12 @@ type AllowArbitraryOperationsHandler = (
   request: Request,
 ) => PromiseOrValue<boolean>
 
-export interface UsePersistedOperationsOptions {
-  /**
-   * A function that fetches the persisted operation
-   */
-  getPersistedOperation(key: string): PromiseOrValue<string | null>
+export enum PersistedOperationType {
+  SDL = 'SDL',
+  AST = 'AST',
+}
```

Comment:
Couldn't we do a `typeof document === "string"` check instead to avoid this additional configuration option?

Reply:
Done 👍